### DISCUSSION TITLE
Option to hide starred repo that already in a list.

### DISCUSSION THREAD

**Discussion by vunhatchuong** [community/community#37746] [Other Feature Feedback, Questions, & Ideas]
Lists are made to organize your starred repo, it would be great if you can hide repo that's already inside a list so my "Your Stars" tab is not a mess.

**Comment by jfullstackdev** [comment]
that can be the reverse too, others might need to star the same repo in different lists as per their liking or requirement

**Comment by ossanna16** [comment]
Hi there @vunhatchuong123 and welcome to our community! Thank you for the feedback!
